components/_global-navigation.scss

```diff
@@ -215,3 +266,26 @@ $notification-counter-size: 18px;
 		}
 	}
 }
+
+
+
+.wds-search-is-active {
+	.wds-global-navigation__search {
```

Comment:
I think it should be where the other styles for this class are, like it was before the last commit. It will be harder to find as it is now.